src/index.ts

```diff
@@ -27,7 +27,13 @@ export {
 	RenditionUiSchema,
 } from './components/Form';
 
-export { Renderer, RendererProps } from './components/Renderer';
+export {
+	Renderer,
+	RendererBase,
```

Comment:
Do we also need the RendererBase?
I mean if we only need the Widget for autoUI, then I would avoid also exporting any *Base components.

Reply:
Sorry this remained from tests. Removing it